src/js/main/show.js

```diff
@@ -153,20 +153,52 @@ const app = new Vue({
   }
 });
 
+const searchBarApp = new Vue({
+  el: "#section-search",
+  data: {
+    search_type: "by-company",
+    keyword: "",
+  },
+  methods: {
+    onSubmit: function() {
+      if (this.search_type === "by-company") {
+        router.setRoute(`/search-and-group/by-company/${encodeURIComponent(this.keyword)}`);
+      } else if (this.search_type === "by-job-title") {
+        router.setRoute(`/search-and-group/by-job-title/${encodeURIComponent(this.keyword)}`);
+      } else {
+        router.setRoute("/latest");
+      }
+    },
+    setDefault: function() {
+      this.search_type = "by-company";
+      this.keyword = "";
+    },
+    setInputInfo: function(search_type, keyword) {
```

Comment:
I don't get it.
It will have error when I change to `function(search_type = "by-company", keyword = "")`